packages/trpc/server/routers/viewer/eventTypes/create.handler.ts

```diff
@@ -41,7 +41,23 @@ export const createHandler = async ({ ctx, input }: CreateOptions) => {
   }
 
   if (defaultConferencingData && defaultConferencingData.appSlug !== "daily-video") {
-    const credentials = ctx.user.credentials;
+    const credentials = await ctx.prisma.credential.findMany({
+      where: {
+        userId: ctx.user.id,
+      },
+      select: {
+        id: true,
+        type: true,
+        key: true,
+        userId: true,
+        appId: true,
+        invalid: true,
+        teamId: true,
+      },
+      orderBy: {
+        id: "asc",
+      },
+    });
```

Comment:
Would love to abstract these repeated calls. Not a blocker tho.

Reply:
I can do this :)